airbyte-integrations/connectors/source-relational-db/src/main/resources/ssh-tunnel-spec.json

```diff
@@ -0,0 +1,167 @@
+{
```

Comment:
is this file the same as the one in `airbyte-integrations/connectors/destinations-relational-db/src/main/resources/ssh-tunnel-spec.json`? Is there any way we can DRY it out?